### DISCUSSION TITLE
A Suggestion: Enable to Disable View Modes...

### DISCUSSION THREAD

**Discussion by Kimet1970** [djyt/cannonball#103] [General]
**A little suggestion...**
The "only highway" view mode is interesting... but hard to drive...
One second on it... and we can to collide!
If you switch from "normal" to "above view" mode... You must necessary to go through the "highway" mode to go back to "normal" mode...
My suggestion is to add in the "config" file something as:

**View_modes activation:
<Normal_vision>1<Normal_vision>
<Above_vision>1<Above_vision>
<Only_Highway_vision>0<Only_Highway_vision>**

*The option to display a graphical (similar or even identical to "MAME gear shifter") would be great too!

**Comment by yepyepyep4711** [comment]
> *The option to display a graphical (similar or even identical to "MAME gear shifter") would be great too!
You have noticed the L and H in the bottom left, though? In the latest versions the H is green, making it easier to differentiate from the L with peripheral vision. But yes I had to get used to the fact that it's not to the same hand side as the clutch in the original.
